c/common/src/codingstandards/c/OutOfBounds.qll

```diff
@@ -0,0 +1,1250 @@
+/**
+ * This module provides classes and predicates for analyzing the size of buffers
+ * or objects from their base or a byte-offset, and identifying the potential for
+ * expressions accessing those buffers to overflow.
+ */
+
+import cpp
+import codingstandards.c.Pointers
+import codingstandards.c.Variable
+import codingstandards.cpp.Allocations
+import codingstandards.cpp.Overflow
+import codingstandards.cpp.PossiblyUnsafeStringOperation
+import codingstandards.cpp.SimpleRangeAnalysisCustomizations
+import semmle.code.cpp.dataflow.DataFlow
+import semmle.code.cpp.valuenumbering.GlobalValueNumbering
+import semmle.code.cpp.security.BufferAccess
```

Comment:
```suggestion
import semmle.code.cpp.security.BufferAccess as StdlibBufferAccess
```

Reply:
I have removed the imports entirely, as they were unnecessary.